test/verify/check-realms

```diff
@@ -381,12 +381,14 @@ class TestRealms(MachineCase):
             export LC_ALL=C.UTF-8 && while ! ipa user-find >/dev/null; do sleep 5; done'
             """, timeout=300)
         # set up an IPA user with a TLS certificate; can't use "admin" due to https://pagure.io/freeipa/issue/6683
-        ipa_machine.execute("""docker exec -i freeipa sh -exc '
+        ipa_machine.execute(r"""docker exec -i freeipa sh -exc '
 CERTUSER=jane
 ipa user-add --first=$CERTUSER --last="developer" $CERTUSER
 yes foobar | ipa user-mod --password $CERTUSER
 ipa user-mod --password-expiration='2030-01-01T00:00:00Z' $CERTUSER
-openssl req -x509 -newkey rsa:2048 -days 365 -nodes -keyout ${CERTUSER}.key -out ${CERTUSER}.pem -subj "/CN=$CERTUSER"
+# browsers insist on "Key Usage" extension
```

Comment:
can you be more specific about which browsers, and which versions?

Reply:
I haven't done much research about that. I know that it worked without this in Firefox 69 (in October), and that Firefox 71 now requires it. I haven't tried other browsers.

Reply:
can we at least agree to write "some browsers" then? :)

(and ditto for the example in the docs)

Reply:
Sure! Pushed.